CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [6.4.0] - 2023-11-??
 
+### Exporter
+- Added: Additional Joke Hint referring to Raven Beak and how he uses the 'But He Sounds Handsome' trope when posing as ADAM.
```

Comment:
i'd just shorten this to "Additional Joke Hint referencing Raven Beak"

Also unsure whether it should be in exporter, or in the metroid prime 2 category, as IIRC the joke hints are currently only used for that game

Reply:
Had a look, and the most recent addition to them was under the Prime 2 heading, so moved it there.